utils/docker/Dockerfile.centos.7

```diff
@@ -180,28 +176,30 @@ RUN if [ "x$BULLSEYE" != "x" ]; then \
       rm -rf bullseye; \
     fi
 
-
 # force an upgrade to get any newly built RPMs
 ARG CACHEBUST=1
 RUN yum -y upgrade --exclude=spdk,spdk-devel,dpdk-devel,dpdk,mercury-devel,mercury
 
-# Switch to new user
-#USER $USER
-WORKDIR /home/$USER
-
-# set NOBUILD to disable git clone & build
-ARG NOBUILD
-
-# Fetch DAOS code
-RUN if [ "x$NOBUILD" = "x" ] ; then \
-    git clone --recurse-submodules https://github.com/daos-stack/daos.git; fi
-WORKDIR /home/$USER/daos
+# copy DAOS tree
+WORKDIR /home/daos
+COPY ./ .
 
 # Build DAOS & dependencies
-RUN if [ "x$NOBUILD" = "x" ] ; then \
-    scons --build-deps=yes install PREFIX=/opt/daos; fi
+RUN scons --build-deps=yes -j 4 install PREFIX=/opt/daos
```

Comment:
do we not need the x$NOBUILD anymore?  Obviously, we use it for checkpatch at least

Reply:
Yeah, i am readding. I was just hoping to move to multi-stage build, but that's not possible yet since docker is too old on CentOS 7.